common/app/templates/inlineJS/blocking/config.scala.js

```diff
@@ -18,6 +18,7 @@ window.guardian = {
     isModernBrowser : isModernBrowser,
     isEnhanced:
         window.shouldEnhance && isModernBrowser,
+    supportsModules: window.guardian && window.guardian.supportsModules !== false,
```

Comment:
Isn't this redundant?

Reply:
It's there because this I'm using the namespace `window.guardian` for the `nomodule` script, which then gets overwritten by this declaration - so I'm just passing it along here - which I agree is not very nice. I guess normally I'd extend this object with `...window.guardian` but that just seems a little fiddly without destructuring and for the sake of one property.